docs/site-planning.md

```diff
@@ -164,9 +164,7 @@ We recommend to use a dedicated server for hosting the shared file system. The e
 
 #### NFS Warning
 
-NFS is known to be an *easy* but never the less %RED%inadequate%ENDCOLOR% choice for all but the smallest sites! If you want to use NFS you should read the chapter *"Optimizing NFS Performance"* in the [http://nfs.sourceforge.net/nfs-howto/ Linux NFS HowTo](http://nfs.sourceforge.net/nfs-howto/ Linux NFS HowTo). Software partitions that can be locally installed, such as OSG\_GRID, should be locally installed and not shared unless you have an enterprise-class NFS server.
-
-Condor provides mechanisms which can be used to greatly minimize NFS usage [NFSLite](http://vdt.cs.wisc.edu/releases/2.0.0/notes/Globus-CondorNFSLite-Setup.html). This package makes it unnecessary to have NFS-mounted **HOME** directories on all the worker nodes. Security is also improved because user proxies will not be sent across NFS in the clear. The running job will only have access to grid or voms proxies running at the same time on the same worker node as the job itself avoiding identity theft.
+NFS is known to be an *easy* but never the less %RED%inadequate%ENDCOLOR% choice for all but the smallest sites! If you want to use NFS you should read the chapter *"Optimizing NFS Performance"* in the <http://nfs.sourceforge.net/nfs-howto/>. Software partitions that can be locally installed, such as OSG\_GRID, should be locally installed and not shared unless you have an enterprise-class NFS server.
```

Comment:
That sentence is easy to parse incorrectly!

I initially read it as "easy but *never* the less-inadequate".  I think you meant "easy but never-the-less inadequate".  Dropping the double-negatives, "easy but less adequate for large sites" would do.